target/linux/ath79/image/generic.mk

```diff
@@ -655,6 +655,17 @@ define Device/rosinson_wr818
 endef
 TARGET_DEVICES += rosinson_wr818
 
+define Device/wd_mynet-n750
+  $(Device/seama)
+  ATH_SOC := ar9344
+  DEVICE_TITLE := Western Digital My Net N750
+  IMAGE_SIZE := 15808k
```

Comment:
0xf80000 = 15872k ?

Reply:
I was following the ar71xx profile here - didn't want to take a chance for the sake of 0.4% extra space.

I've tested with 15872k and it works fine, inc. sysupgrade from ar71xx preserving config.

Reply:
Well, ar71xx could be wrong, I've just checked IMAGE_SIZE as it should correspond with the firmware partition size otherwise it simply doesn't make sense and might be confusing and should be probably noted in the commit message.